### ISSUE TITLE
MD070: Be able to enable rule in its own configuration block

### ISSUE THREAD

**Issue by eread** [rvben/rumdl#520]
### Problem or motivation

Rule MD070 (https://github.com/rvben/rumdl/blob/a304d288c836b694e792d329a782db3e3165858c/docs/md070.md) was recently documented as opt-in: https://github.com/rvben/rumdl/commit/0cc7f9f3ad529544ff491dfcf6fc7bec2fddc7e7.

In addition to being able to enable rules globally using `enable` or `extend-enable` https://github.com/rvben/rumdl/blob/a304d288c836b694e792d329a782db3e3165858c/docs/global-settings.md#quick-reference), rules can also be enabled in their own block. For example, MD063: https://github.com/rvben/rumdl/blob/a304d288c836b694e792d329a782db3e3165858c/docs/md063.md#enable-with-title-case-default.

However, MD070 can't be enabled in its own configuration block. If one tries, `rumdl` returns:

```shell
[config warning] Unknown option for rule MD070: enabled
```

### Proposed solution

I wonder if it should be possible to enable MD070 with this configuration:

```toml
[MD070]
enabled = true
```

### Alternatives considered

I'd wondered if rules that don't expose any other configuration would only be enabled in `global` configuration. Perhaps MD063 can be enabled in its own configuration block because it also exposes other settings?

However, it makes sense to me that even for rules that can only be enabled and disabled, that it would be possible to enable them _either_ in `global` configuration or in their own configuration block. It would be a personal preference.

### Feature category

CLI / configuration

**Comment by rvben** [comment]
Fixed in v0.1.46. The `enabled` key is now accepted in any rule's config block without unknown-key warnings. Additionally, `enabled = true/false` is now fully bidirectional — `enabled = false` actually disables the rule, and per-rule `enabled` takes precedence over global `disable`/`extend-disable`/`extend-enable` lists. Thanks for reporting @eread!
